src/react-components/src/components/01-atoms/Links/LinkHelper.tsx

```diff
@@ -0,0 +1,57 @@
+// Base Link Helper Class
+import * as React from "react";
+
+export interface LinkHelperProps {
+  className: string;
+  icon?: { element: JSX.Element };
+  url?: string;
+  attributes?: {};
+}
+
+export default class LinkHelper extends React.Component<LinkHelperProps, {}> {
+  constructor(props: LinkHelperProps) {
+    super(props);
+  }
+
+  render(): JSX.Element {
+    const { url, className, attributes, icon } = this.props;
+
+    let props = {
+      className: className,
+      ...attributes
+    };
+
+    if (React.Children.count(this.props.children) > 1) {
+      // There should only be one child
+      // but React.Children.only() can't cast to React.ReactElement for typechecking
+      // Therefore, React.Children.map() must be used.
+
+      throw new Error(`Please only pass one child, got ${this.props.children}`);
+    }
+
+    let link = React.Children.map(this.props.children, (child: React.ReactElement, index) => {
+      if (child.type !== "a") {
```

Comment:
This rejects React.Router `<Link>` elements, which moots the point.  Looking into it again